components/Frame/Header.js

```diff
@@ -131,24 +134,61 @@ const styles = {
       paddingTop: '18px'
     },
     transition: 'opacity .2s ease-in-out'
+  }),
+  sticky: css({
+    position: 'sticky'
+  }),
+  stickyWithFallback: css({
+    position: ['fixed', 'sticky']
+  }),
+  hr: css({
+    margin: 0,
+    display: 'block',
+    border: 0,
+    color: colors.divider,
+    backgroundColor: colors.divider,
+    width: '100%',
+    zIndex: ZINDEX_HEADER
+  }),
+  hrThin: css({
+    height: 1,
+    top: HEADER_HEIGHT_MOBILE - 1,
```

Comment:
On Chrome/Android I see a bit of white under the thin HR:
<img width="1098" alt="screen shot 2018-08-23 at 11 50 33" src="https://user-images.githubusercontent.com/23520051/44518626-0bb84e00-a6cb-11e8-8755-c20ef46e4eca.png">